mmv1/products/apigee/terraform.yaml

```diff
@@ -161,6 +161,30 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       delete_minutes: 30
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       custom_import: templates/terraform/custom_import/apigee_instance_attachment.go.erb
+  EnvgroupAttachment: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: true
+    import_format: ["{{envgroup_id}}/attachments/{{name}}", "{{envgroup_id}}/{{name}}"]
+    delete_url: '{{envgroup_id}}/attachments/{{name}}'
```

Comment:
Can we also add a `skip_sweeper` override? This resource depends on an Environment ID, and our generated sweepers cannot handle that. Like so: https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/products/sql/terraform.yaml#L26